### ISSUE TITLE
Suggestion: allow a glob pattern as input, and a target directory as output

### ISSUE THREAD

**Issue by msikma** [shakiba/svgexport#6]
This would allow the script to be used for converting large numbers of SVGs. I had to write a quick script for doing this just now.

**Comment by shakiba** [comment]
Yes, this is common use case, just few days ago I was thinking about a similar idea. But I doubt that there is anyway to handle it cleanly within this library, there are many different use cases and corner cases and it looks out of scope of this library. I think there are better options to handle this, for example you can use `find -exec`.

Currently it is also possible to pass a json file name which includes input/outputs or a js file name which `module.exports` similar contents. Additionally it may be possible to implement a new way to pipe-in this content, instead of passing a file name, so that you can dynamically generate it and pass it to svgexport.

**Comment by msikma** [comment]
Would it be sufficient to just add the option to use a glob pattern for input, and a target directory for output? The caveat would be that you cannot pass specific options per file. But it seems doable. I could make a PR for it sometime this weekend.

**Comment by shakiba** [comment]
I think its better to find a way to handle this kind of use cases instead of only this one, for example someone else may need to use regex for file selection or read input list from a file. For output as you mentioned providing only a directory is not enough, you also need output options and also output file name.  Moreover adding glob would diverge this library from its core functionality and makes it difficult to maintain.

**Comment by tannerhodges** [comment]
For anyone else wanting to do this now, here's the bash script I'm using:

```
find . -name '*.svg' -exec sh -c 'svgexport "${0%}" "${0%.*}.png"' {} \;
```

If you need to export to another directory just add it in (e.g., a `png/` subdirectory):

```
find . -name '*.svg' -exec sh -c 'svgexport "${0%}" png/"${0%.*}.png"' {} \;
```

And for those with a more complex setup, you can strip the full path from the filename and store it in a var (e.g., `name`):

```
find . -name '*.svg' -exec sh -c 'name=`basename "${0%}"`; svgexport "${name%}" png/"${name%.*}".png 2x' {} \;
```

**Comment by shakiba** [comment]
@tannerhodges Thank you!

**Comment by shakiba** [comment]
@msikma I have decided to implement this, if you are interested please make a PR.

**Comment by msikma** [comment]
I'm a bit busy lately... I'm not sure I'll be able to do this for a while unfortunately. Maybe in the future.

**Comment by theogravity** [comment]
You might want to also try https://github.com/theogravity/svg-rasterizer
